### PR TITLE
Update Portugese Brazilian (pt-br) Foundational docs bucket

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update Portuguese Brazilian (pt-br) Foundational docs bucket

### CODE REVIEW

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+description: Uma visão geral dos nós de arquivo
+lang: pt-br
+sidebarDepth: 2
+---
+
+Um nó de arquivo é uma instância de um cliente Ethereum configurado para construir um arquivo de todos os estados históricos. É uma ferramenta útil para certos casos de uso, mas pode ser mais complicado de executar do que um nó completo.
+
+## Pré-requisitos {#prerequisites}
+
+Você deve entender o conceito de um [nó Ethereum](/developers/docs/nodes-and-clients/), [sua arquitetura](/developers/docs/nodes-and-clients/node- arquitetura/), [estratégias de sincronização](https://ethereum.org/en/developers/docs/nodes-and-clients/#sync-modes), práticas de [ execução](/developers/docs/nodes-and-clients/run-a-node/) e [uso deles](/developers/docs/apis/json-rpc/).
```
Comment (minimalsm) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md:12]:
```suggestion
Você deve entender o conceito de um [nó Ethereum](/developers/docs/nodes-and-clients/), [sua arquitetura](/developers/docs/nodes-and-clients/node- arquitetura/), [estratégias de sincronização](https://ethereum.org/en/developers/docs/nodes-and-clients/#sync-modes), práticas de [execução](/developers/docs/nodes-and-clients/run-a-node/) e [uso deles](/developers/docs/apis/json-rpc/).
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ O Ethereum pós Fusão consiste em duas partes: a camada de execução e a camad
...
 
-Antes da [Fusão](/roadmap/merge/), a camada de consenso e execução eram redes separadas, com todas as transações e atividades do usuário no Ethereum acontecendo, no que é agora a camada de execução. Um software do cliente forneceu ambiente de execução e verificação de consenso de blocos produzidos por mineradores. A camada de consenso, [a Beacon Chain](/roadmap/beacon-chain/), está sendo executada separadamente desde dezembro de 2020. Ela introduziu a prova de participação e coordenou a rede de validadores com base nos dados da rede Ethereum.
+Antes da [Fusão](/upgrades/merge/), a camada de consenso e execução eram redes separadas, com todas as transações e atividades do usuário no Ethereum acontecendo, no que é agora a camada de execução. Um software do cliente forneceu ambiente de execução e verificação de consenso de blocos produzidos por mineradores. A camada de consenso, [a Beacon Chain](/upgrades/beacon-chain/), está sendo executada separadamente desde dezembro de 2020. Ela introduziu a prova de participação e coordenou a rede de validadores com base nos dados da rede Ethereum.
```
Comment (minimalsm) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/index.md:27]:
```suggestion
Antes da [Fusão](/roadmap/merge/), a camada de consenso e execução eram redes separadas, com todas as transações e atividades do usuário no Ethereum acontecendo, no que é agora a camada de execução. Um software do cliente forneceu ambiente de execução e verificação de consenso de blocos produzidos por mineradores. A camada de consenso, [a Beacon Chain](/roadmap/beacon-chain/), está sendo executada separadamente desde dezembro de 2020. Ela introduziu a prova de participação e coordenou a rede de validadores com base nos dados da rede Ethereum.
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/nodes-as-a-service/index.md [hunk 3]
```diff
@@ -315,15 +315,27 @@ Segue uma lista de alguns dos fornecedores de nós para Ethereum mais populares.
...
+  - Usamos roteamento inteligente para sempre atingir o nó mais próximo de seu usuário
+
 ## Leitura adicional {#further-reading}
 
 - [Lista dos serviços de nós Ethereum](https://ethereumnodes.com/)
 
 ## Tópicos relacionados {#related-topics}
 
-- [Nós e clientes](/developers/docs/nodes-and-clients/)
+- [ Nós e clientes](/developers/docs/nodes-and-clients/)
```
Comment (minimalsm) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/nodes-as-a-service/index.md:336]:
```suggestion
- [Nós e clientes](/developers/docs/nodes-and-clients/)
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+description: Uma visão geral dos nós de arquivo
+lang: pt-br
+sidebarDepth: 2
+---
+
+Um nó de arquivo é uma instância de um cliente Ethereum configurado para construir um arquivo de todos os estados históricos. É uma ferramenta útil para certos casos de uso, mas pode ser mais complicado de executar do que um nó completo.
+
+## Pré-requisitos {#prerequisites}
+
+Você deve entender o conceito de um [nó Ethereum](/developers/docs/nodes-and-clients/), [sua arquitetura](/developers/docs/nodes-and-clients/node- arquitetura/), [estratégias de sincronização](https://ethereum.org/en/developers/docs/nodes-and-clients/#sync-modes), práticas de [execução](/developers/docs/nodes-and-clients/run-a-node/) e [uso deles](/developers/docs/apis/json-rpc/).
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md:12]:
```suggestion
Você deve entender o conceito de um [nó Ethereum](/developers/docs/nodes-and-clients/), [sua arquitetura](/developers/docs/nodes-and-clients/node- arquitetura/), [estratégias de sincronização](/developers/docs/nodes-and-clients/#sync-modes), práticas de [execução](/developers/docs/nodes-and-clients/run-a-node/) e [uso deles](/developers/docs/apis/json-rpc/).
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+Antes de iniciar seu próprio nó de arquivo, aprenda sobre as diferenças entre os clientes e especialmente os vários [requisitos de hardware](/developers/docs/nodes-and-clients/run-a-node/#requirements). A maioria dos clientes não é otimizada para esse recurso e seus arquivos exigem mais de 12 TB de espaço. Por outro lado, implementações como Erigon podem armazenar os mesmos dados em menos de 3 TB, o que as torna a forma mais eficaz de executar um nó de arquivo.
+
+## Práticas recomendadas
+
+Além das [recomendações gerais para executar um nó](developers/docs/nodes-and-clients/run-a-node/), um nó de arquivo pode demandar mais hardware e manutenção. Considerando as [principais funcionalidades](https://github.com/ledgerwatch/erigon#key-features) do Erigon, a abordagem mais prática é usar a implementação cliente do [Erigon](https://ethereum.org /en/developers/docs/nodes-and-clients/#erigon).
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md:59]:
```suggestion
Além das [recomendações gerais para executar um nó](developers/docs/nodes-and-clients/run-a-node/), um nó de arquivo pode demandar mais hardware e manutenção. Considerando as [principais funcionalidades](https://github.com/ledgerwatch/erigon#key-features) do Erigon, a abordagem mais prática é usar a implementação cliente do [Erigon](/developers/docs/nodes-and-clients/#erigon).
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/node-architecture/index.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+title: Arquitetura do nó
+description: Introdução de como os nós Ethereum são organizados.
+lang: pt-br
+---
+
+Um nó Ethereum é composto por dois clientes: um [cliente de execução](/developers/docs/nodes-and-clients/#execution-clients) e um [cliente de consenso](/developers/docs/ nodes-and-clients/#consensus-clients).
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/node-architecture/index.md:7]:
```suggestion
Um nó Ethereum é composto por dois clientes: um [cliente de execução](/developers/docs/nodes-and-clients/#execution-clients) e um [cliente de consenso](/developers/docs/nodes-and-clients/#consensus-clients).
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/node-architecture/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+Quando o Ethereum estava usando [prova de trabalho](/developers/docs/consensus-mechanisms/pow/), um cliente de execução era suficiente para executar um nó completo Ethereum. No entanto, desde a implementação da [prova de participação](/developers/docs/consensus-mechanisms/pow/), o cliente de execução precisa ser usado com outro software, chamado [“cliente de consenso”](/developers/docs/nodes-and-clients/#consensus-clients).
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/node-architecture/index.md:9]:
```suggestion
Quando o Ethereum estava usando [prova de trabalho](/developers/docs/consensus-mechanisms/pow/), um cliente de execução era suficiente para executar um nó completo Ethereum. No entanto, desde a implementação da [prova de participação](/developers/docs/consensus-mechanisms/pos/), o cliente de execução precisa ser usado com outro software, chamado [“cliente de consenso”](/developers/docs/nodes-and-clients/#consensus-clients).
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/node-architecture/index.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+O diagrama abaixo mostra a relação entre os dois clientes Ethereum. Os dois clientes se conectam às suas respectivas redes ponto a ponto (P2P). As redes P2P separadas são necessárias à medida que os clientes de execução espalham transações em sua rede P2P, permitindo que eles gerenciem seu pool de transações local, enquanto os clientes de consenso distribuem blocos em sua rede P2P, permitindo consenso e crescimento da cadeia.
+
+![](node-architecture-text-background.png)
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/node-architecture/index.md:13]: missing `alt` prop

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/nodes-as-a-service/index.md [hunk 9]
```diff
@@ -325,5 +337,5 @@ Segue uma lista de alguns dos fornecedores de nós para Ethereum mais populares.
 
 ## Tutoriais relacionados {#related-tutorials}
 
-- [Introdução ao desenvolvimento de Ethereum usando Alquimia](/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/)
+- [Introdução ao desenvolvimento do Ethereum usando o Alquimia](/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/)
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/nodes-as-a-service/index.md:340]:
```suggestion
- [Introdução ao desenvolvimento do Ethereum usando o Alchemy](/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/)
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/nodes-as-a-service/index.md [hunk 10]
```diff
@@ -325,5 +337,5 @@ Segue uma lista de alguns dos fornecedores de nós para Ethereum mais populares.
...
 - [Guia para enviar transações usando web3 e Alquimia](/developers/tutorials/sending-transactions-using-web3-and-alchemy/)
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/nodes-as-a-service/index.md:341]:
```suggestion
- [Guia para enviar transações usando web3 e Alchemy](/developers/tutorials/sending-transactions-using-web3-and-alchemy/)
```

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/run-a-node/index.md [hunk 11]
```diff
@@ -460,11 +462,10 @@ Como parte de seu monitoramento, certifique-se de ficar de olho no desempenho de
 - [Guias de participação do Ethereum](https://github.com/SomerEsat/ethereum-staking-guides) - _Somer Esat, atualizado com frequência_
 - [Guia | Como configurar um validador para participação do Ethereum na rede principal](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet) _– CoinCashew, atualizado regularmente_
 - [Guias do ETHStaker sobre como executar validadores em redes de teste](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, atualizado regularmente_
-- [Guias de Participação do Ethereum](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, atualizado regularmente_
-- [FAQ sobre A Fusão para operadores de nós](https://notes.ethereum.org/@launchpad/node-faq-merge) - _julho de 2022_
+- [Perguntas frequentes sobre o The Merge para operadores de nós](https://notes. ethereum. org/@launchpad/node-faq-merge) — _julho de 2022_
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/run-a-node/index.md:465]:
```suggestion
- [Perguntas frequentes sobre o The Merge para operadores de nós](https://notes.ethereum.org/@launchpad/node-faq-merge) — _julho de 2022_
```

**Review by nhsz — CHANGES_REQUESTED** [review]
Thanks, requesting changes to some links

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+description: Uma visão geral dos nós de arquivo
+lang: pt-br
+sidebarDepth: 2
+---
+
+Um nó de arquivo é uma instância de um cliente Ethereum configurado para construir um arquivo de todos os estados históricos. É uma ferramenta útil para certos casos de uso, mas pode ser mais complicado de executar do que um nó completo.
+
+## Pré-requisitos {#prerequisites}
+
+Você deve entender o conceito de um [nó Ethereum](/developers/docs/nodes-and-clients/), [sua arquitetura](/developers/docs/nodes-and-clients/node- arquitetura/), [estratégias de sincronização](/developers/docs/nodes-and-clients/#sync-modes), práticas de [execução](/developers/docs/nodes-and-clients/run-a-node/) e [uso deles](/developers/docs/apis/json-rpc/).
```
Comment (corwintines) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/archive-nodes/index.md:12]:
```suggestion
Você deve entender o conceito de um [nó Ethereum](/developers/docs/nodes-and-clients/), [sua arquitetura](/developers/docs/nodes-and-clients/node-architecture/), [estratégias de sincronização](/developers/docs/nodes-and-clients/#sync-modes), práticas de [execução](/developers/docs/nodes-and-clients/run-a-node/) e [uso deles](/developers/docs/apis/json-rpc/).
```

**Review by nhsz — APPROVED** [review]

File: src/content/translations/pt-br/developers/docs/nodes-and-clients/run-a-node/index.md [hunk 13]
```diff
@@ -460,11 +462,10 @@ Como parte de seu monitoramento, certifique-se de ficar de olho no desempenho de
...
+- [Perguntas frequentes sobre o The Merge para operadores de nós](https://notes.ethereum.org/@launchpad/node-faq-merge) — _julho de 2022_
 - [Analisando os requisitos de hardware para ser um nó totalmente validado no Ethereum](https://medium.com/coinmonks/analyzing-the-hardware-requirements-to-be-an-ethereum-full-validated-node-dc064f167902) _ – Albert Palau, 24 de setembro de 2018_
-- [Executando nós completos Ethereum: Um guia para os pouco motivados](https://medium.com/@JustinMLeroux/running-ethereum-full-nodes-a-guide-for-the-barely-motivated-a8a13e7a0d31) _ – Justin Leroux, 7 de novembro de 2019_
-- [Executando um nó do Hyperledger Besu na Rede principal do Ethereum: benefícios, requisitos e configurações](https://pegasys.tech/running-a-hyperledger-besu-node-on-the-ethereum-mainnet-benefits-requirements-and-setup/) _– Felipe Faraggi, 7 de maio de 2020_
+- [Executando nós completos do Ethereum: um guia para os pouco motivados ](https://medium.com/@JustinMLeroux/running-ethereum-full-nodes-a-guide-for-the-barely-motivated-a8a13e7a0d31) _– Justin Leroux, 7 de novembro de 2019_
```
Comment (corwintines) [inline src/content/translations/pt-br/developers/docs/nodes-and-clients/run-a-node/index.md:467]:
```suggestion
- [Executando nós completos do Ethereum: um guia para os pouco motivados](https://medium.com/@JustinMLeroux/running-ethereum-full-nodes-a-guide-for-the-barely-motivated-a8a13e7a0d31) _– Justin Leroux, 7 de novembro de 2019_
```

**Review by corwintines — CHANGES_REQUESTED** [review]

**Review by corwintines — APPROVED** [review]

**Pull request merged**